Binary files orig/proj4/src/.Makevars.ucrt.swp and patched/proj4/src/.Makevars.ucrt.swp differ
diff -Nru orig/proj4/src/Makevars.ucrt patched/proj4/src/Makevars.ucrt
--- orig/proj4/src/Makevars.ucrt	2023-06-13 06:54:03.000000000 +0200
+++ patched/proj4/src/Makevars.ucrt	2023-10-10 09:16:59.453455500 +0200
@@ -1,4 +1,8 @@
 ## from Tomas, uses P6 API so verify on toolchain updates
-LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
-PKG_LIBS = -lproj -lsqlite3 -lcurl -lbcrypt -ltiff -ljpeg -lrtmp -lssl -lssh2 -lgcrypt -lcrypto -lgdi32 -lz -lzstd -lwebp $(LIBSHARPYUV) -llzma -lgdi32 -lcrypt32 -lidn2 -lunistring -liconv -lgpg-error -lws2_32 -lwldap32 -lwinmm -lstdc++
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
+  PKG_LIBS = -lproj -lsqlite3 -lcurl -lbcrypt -ltiff -ljpeg -lrtmp -lssl -lssh2 -lgcrypt -lcrypto -lgdi32 -lz -lzstd -lwebp $(LIBSHARPYUV) -llzma -lgdi32 -lcrypt32 -lidn2 -lunistring -liconv -lgpg-error -lws2_32 -lwldap32 -lwinmm -lstdc++
+else
+  PKG_LIBS = $(shell pkg-config --libs proj)
+endif
 PKG_CPPFLAGS = -DUSE_PROJ6_API=1
